docs/pages/troubleshooting/clear-cache-macos-linux.mdx

```diff
@@ -1,5 +1,6 @@
 ---
 title: Clearing bundler caches on macOS and Linux
+description: Learn about the measures you can take to clear bundler caches on macOS and Linux when using Expo.
```

Comment:
I don't like that much the `measures you can take` part, it seems to make description less concise and the term itself do not add any information. But in the other hand, description without this addition is quite similar to the page title.

Another part is the "with Expo" suffix, looking at the page content, we also guide people how to clear the project when they are not using Expo, but RN CLI.

Below are two suggestion, but feel free to alter the description in other way, those are just examples.

```suggestion
description: Learn how to clear bundler caches on macOS and Linux when building React Native app.
```

```suggestion
description: Learn how to clear React Native app bundler caches on macOS and Linux.
```

Reply:
Great points!

I didn't give it much of thought initially. Have removed the phrasing you pointed out from all the pages.

Reply:
@Simek, I've updated the description tag for the pages under the `/troubleshooting/`. I've also fixed formatting issues (such as not using `Terminal` component) and some minor grammatical issues.